specification/maps/data-plane/Creator/preview/2022-01-01-preview/features.json

```diff
@@ -0,0 +1,964 @@
+        "operationId": "Features_GetFeatures",
+        "x-ms-examples": {
+          "GetFeatures": {
+            "$ref": "./examples/features/Features_GetFeatures.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../2.0/dataset.json#/parameters/DatasetId"
+          },
+          {
+            "$ref": "#/parameters/CollectionId"
+          },
+          {
+            "$ref": "#/parameters/LimitFeatures"
+          },
+          {
+            "$ref": "#/parameters/Bbox"
+          },
+          {
+            "$ref": "#/parameters/Filter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/ExtendedGeoJsonFeatureCollection"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      },
+      "post": {
+        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nCreator makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. [This](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps) article introduces concepts and tools that apply to Azure Maps Creator. For more information with regards to ontology, please read this [article](https://docs.microsoft.com/azure/azure-maps/creator-facility-ontology?pivots=facility-ontology-v2) describing the Facility ontology.\n\n The Features API is part of Creator. Features API follows the [Open Geospatial Consortium API standard for Features](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html) to query [Datasets](https://docs.microsoft.com/en-us/rest/api/maps/v2/dataset/create). \nA dataset consists of multiple feature collections. A feature collection is a collection of features of a similar type, based on a common schema.\nThe Post Feature API creates a feature with the provided attributes in the given collection.",
```

Comment:
Should we take another look at this boilerplate description? Is it really adding anything?

Reply:
I'm personally a fan of the links in the description since they provide useful references if it's your first time reading about the service. I didn't want to rock the boat too much here, but I'm definitely open to suggestions!

Reply:
Having the links here is useful in the scenario that someone searches for one of the APIs and wants to get to learn more about the Creator service.  However, I think that this is something that should be updated a bit, also maybe changing the order of things so that the overview page has content that is more useful.

Reply:
Great suggestion on updating the order so that the overview page is more useful. I've now cleaned up and updated the description for all the Features APIs. Let me know what your thoughts are.